### ISSUE TITLE
CLI: Fix sort order bug when saving fields

### ISSUE THREAD

**Issue by jgambarios** [dotCMS/core#29086]
### Parent Issue

https://github.com/dotCMS/core/issues/28880

### Task

When saving fields from the CLI, there is a bug changing the sort order. Even when the sort order is defined properly in the content type descriptor, there is a logic when saving fields that “moves” fields, altering the sort order.

Steps to Reproduce:

1. Define the sort order in the content type descriptor.
2. Save fields using the CLI.
3. Observe that the sort order is altered, breaking the layout in the edit content type portlet.

### Proposed Objective

Same as Parent Issue

### Proposed Priority

Same as Parent Issue

### Acceptance Criteria

The sort order provided in the descriptor should be preserved.

**Comment by dcolina** [comment]
Internal QA: Passed.

This feature works as expected.

We have created a new content-type (named ContentOrder), which one was defined in the following `json` file. It contains several rows and different elements but they have been sorted changing without respecting their statement order. Instead we use the key `order` into each element to indicate how they must be sorted.

JSON file sample.
[ContentOrder.json](https://github.com/user-attachments/files/16509428/ContentOrder.json)

After pushing this new content-type through `dotCLI` tool, we have noticed the content-type have been created  sorting the elements in the way we have been defined into de descriptor.

![Image](https://github.com/user-attachments/assets/974a4938-56bf-4e7e-9d5a-520d77874af1)
